src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -968,6 +978,118 @@ class spell_dru_t4_2p_bonus : public SpellScriptLoader
         }
 };
 
+// 783 - Travel Form
+class spell_dru_travel_form : public SpellScriptLoader
+{
+public:
+    spell_dru_travel_form() : SpellScriptLoader("spell_dru_travel_form") { }
+
+    class spell_dru_travel_form_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_dru_travel_form_SpellScript);
+
+        bool Load()
+        {
+            return GetCaster()->GetTypeId() == TYPEID_PLAYER;
+        }
+
+        SpellCastResult CheckCast()
```

Comment:
Does CheckCast() need override? None of the other CheckCast() declarations in this script use override.